src/styles.scss

```diff
@@ -111,3 +111,53 @@ body {
   padding: 16px;
   width: 100%;
 }
+// how to use it: <button mat-button class="mat-default">button</button>
+// <button mat-button class="mat-default__danger--disabled">button</button>
+.mat-default {
+  background-color: mat.get-color-from-palette(palette.$blue-palette, 500);
+  border: none;
+  color: mat.get-color-from-palette(palette.$gray-palette, 100);
+  padding: 8px 16px;
+  text-align: center;
+  display: inline-block;
+  margin: 8px;
+  border-radius: 8px;
+  &--disabled {
+    @extend .mat-default;
+    background-color: mat.get-color-from-palette(palette.$blue-palette, 100);
+    pointer-events: none;
+  }
+  &__round {
```

Comment:
To według bem mogłoby być modyfikatorem, bo posiada wszystkie style `mat-default`